components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -94,18 +94,30 @@
                 <div class="healthcheck-header {{service.health_check}}">
                   Health Check Error Message
                 </div>
-                <div class="healthcheck-content">
+                <div class="healthcheck-content" [class.active]="activeErrorAccordions.includes(i)">
```

Comment:
We attach an active class here if this particular accordion is included in the active accordions array.